package.json

```diff
@@ -45,7 +45,7 @@
         "@rollup/plugin-replace": "^5.0.7",
         "@rollup/plugin-typescript": "^11.1.6",
         "@types/babel__core": "^7.20.5",
-        "@types/node": "^20.14.11",
+        "@types/node": "^22.1.0",
```

Comment:
I know ideally we should have types for our minimum supported Node version, but right now we don't really declare a minimum supported Node version, so I'm fine with just bumping this.